src/app.ts

```diff
@@ -1 +1,103 @@
+import axios from 'axios';
+import Toastify from 'toastify-js';
 
+const modal = document.querySelector('.dialog') as HTMLDivElement;
+const game = document.querySelector('.js-app') as HTMLDivElement;
+const gameContainer = document.querySelector('.js-game') as HTMLDivElement;
+const nameInput = document.querySelector('.js-input-name') as HTMLInputElement;
+// Buttons
+const playButton = document.querySelector(
+	'.js-play-button'
+) as HTMLButtonElement;
+const restartButton = document.querySelector(
+	'.js-restart-game'
+) as HTMLButtonElement;
+// Hidden class
+const hiddenClass: string = 'hidden';
+// URL
+const API_KEY = '28be1067';
+const API_LINK = 'http://www.omdbapi.com/?t=Star+Wars&apikey=';
+const URL = API_LINK + API_KEY;
+
+let numberOfImage: number;
+
+const closeModal = () => {
+	if (nameInput.value) {
+		modal.classList.add(hiddenClass);
+		game.classList.remove(hiddenClass);
+	} else {
+		Toastify({
+			text: 'Please add your nickname',
+			duration: 3000,
+			newWindow: true,
+			close: true,
+			gravity: 'top', // `top` or `bottom`
+			position: 'center', // `left`, `center` or `right`
+			stopOnFocus: true, // Prevents dismissing of toast on hover
+			style: {
+				background: 'linear-gradient(to right, #00b09b, #96c93d)',
+			},
+			onClick: function () {}, // Callback after click
+		}).showToast();
```

Comment:
Całe to, wyniósłbym nawet do osobnego pliku `helpers` gdzie opakowałbym to w funkcję która będzie wywołwana w wielu miejscach. Powiadomimy usera o braku danych, o starcie gry, o koncu gry a nawet, mozemy go poinformować o błędzie API do pobierania zdjęć (Wtedy mozemy go przenieść znowu do stanu poczatkowego gry z ekranem wyboru poziomu).

Zrobiłbym funkcję showToast która przyjmie parametr tekstu oraz wariant np success, error, warning, i w tej funkcji juz stworzysz konkretne style które określają wariant, dodasz przekazany tekst a reszta która jestniezmienna będzie zdefiniowana tylko raz.